mos-platform/geos-cbm/vlir.ld

```diff
@@ -0,0 +1,218 @@
+__overlaysize   = DEFINED(__overlaysize_param) ? __overlaysize_param : 0x1000;
+
+__backbufsize	= 0x2000;
+__overlayaddr 	= 0x8000 - __backbufsize - __overlaysize;
+__stacksize   	= 0x0400;
+__stack   	= __overlayaddr;
+__zp_reg_start  = 0x90;
+__zp_reg_end    = 0xfb;
+__zp_reg_size   = __zp_reg_end - __zp_reg_start;
+
+MEMORY {
+  zp          : ORIGIN = __zp_reg_start, LENGTH = __zp_reg_size
+  vlir00(rwx) : ORIGIN = 0x0400, LENGTH = __overlayaddr - __stacksize - 0x0400 
```

Comment:
There isn't actually much use to giving more than one of these `(rwx)`; the linker script semantics assign all matching sections to the first match, regardless of whether or not they actually fit. If they don't fit, the link fails. So this mechanism is of very limited utility, and it's usually better to assign sections to memory regions explicity.